tensorflow/lite/experimental/kernels/unidirectional_sequence_gru.cc

```diff
@@ -0,0 +1,173 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/lite/c/c_api_internal.h"
+#include "tensorflow/lite/experimental/kernels/gru_cell.h"
+#include "tensorflow/lite/kernels/internal/tensor.h"
+#include "tensorflow/lite/kernels/kernel_util.h"
+
+namespace tflite {
+namespace ops {
+namespace experimental {
+namespace unidirectional_sequence_gru {
+namespace {
+
+void LoopFloat(const TfLiteTensor* input, const TfLiteTensor* state,
+               const TfLiteTensor* gate_weight, const TfLiteTensor* gate_bias,
+               const TfLiteTensor* candidate_weight,
+               const TfLiteTensor* candidate_bias, TfLiteTensor* output,
+               TfLiteTensor* activation, TfLiteTensor* concat) {
+  const int n_time = input->dims->data[0];
+  const int n_batch = input->dims->data[1];
+  const int n_input = input->dims->data[2];
+  const int n_output = output->dims->data[2];
+  const RuntimeShape input_shape = RuntimeShape({n_batch, n_input});
```

Comment:
nit: may just the constructor is fine

Reply:
I mean you can do `const RuntimeShape input_shape({n_batch, n_input});`

Reply:
Done